http/auth.go

```diff
@@ -18,8 +18,8 @@ import (
 	"github.com/lestrrat-go/jwx/v2/jwt"
 	"github.com/sourcenetwork/immutable"
 
+	"github.com/sourcenetwork/defradb/acp/identity"
 	acpIdentity "github.com/sourcenetwork/defradb/acp/identity"
```

Comment:
todo: You're importing the same package twice under a different name. Please choose only one. (My preference is to just use "identity" instead of "acpIdentity")

Reply:
nitpick: I prefer `acpIdentity` haha and use `identity` for var names rather than `ident`

Reply:
done